### ISSUE TITLE
Reply to deleted issue #17

### ISSUE THREAD

**Issue by benjaminlaib** [ekkash/movieo#1]
I was just going to reply to your comment, however then I realized that it's gone already. Here's what I wrote:

Yes, I got a message from GitHub and apparently, the flagging was the fault of some buggy anti-spam script. I didn't even know those existed, although, afterwards it's pretty logical :D
Anyways...

I'm really sorry to hear that this project is dead. I may be interested in maintaining it (probably also rewriting it to be cross-platform), if you are okay with that. Do you want to get in contact in a more personal way before that (some online chat should do the job lol)? There are some things I am very curious about.
You could use mega.nz to upload the files, it gives you 50 gigs of storage for free and is fast and reliable. There is also a library for downloading files from MEGA, I used it a couple of times and it works much better than dropbox, imho.

I could use the codebase in case I happen to maintain the project in future, that would be awesome. Thank you very much for your kind support!

**Comment by ekkash** [comment]
Of course, I appreciate any contributions and forks of the project, it's not a problem. You can contact me at my email found on my user profile, I'm always up for the chat. I have successfully fixed all the issues along with thr database files added to repo. There's still a couple of ammendments I need to make before I push a release but expect to see one within the next couple of days.

Kind regards, Ashley
